modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -327,16 +329,48 @@ The following list shows the reserved Mule keywords to avoid when naming valid i
 * `target`
 * `targetValue`
 
+[[tls-support]]
+== Generating Connectors With TLS Support
+
+Mule 4 (SmartConnector) connectors support Transport Layer Security (TLS). When you enable TLS support and generate the connector, the connector is upgraded to run on Mule runtime 4.6. and TLS is enabled so that Anypoint Studio displays the appropriate form to collect required TLS connection configuration information for the connector. 
+
+To enable TLS, add a `properties.json` file to the root directory of your Design Center API project for your app: 
+
+. In Design Center, Select *Files +* > *New file*.  
+. In the *Add new file* window, select *API format > other*, name the file `properties.json`, then click *Create*.
+. In the new file, add `"tlsEnabled": true`.
++
+[NOTE]
+Add the `properties.json` file only if you want to enable TLS support or explicitly opt out of it. If you don't add a `properties.json` file to your API project, the REST Connect Service generates the standard connector artifacts, without TLS support.
+
+After you add the `properties.json` file and enable TLS, Anypoint Studio displays the TLS configuration form for the connector.  
+ 
+When you add a `properties.json` file to your project, ensure that the `tlsEnabled` property is correctly set with a valid boolean value of either `true` to enable TLS or `false` to explicitly disable TLS. If the `properties.json` file is invalid or if the property is incorrectly set, the connector generation is aborted.
+
+[[upgrade-rest-connector]]
+== Upgrading a REST Connect Generated Connector
 
+To update your REST Connect generated connector to use the latest version of Mule runtime and to be compatible with Java 17:
+
+. In Design Center, open your API project. 
+. Click *Publish*. 
++
+The connector is re-generated. 
+
+If you want to add <<tls-support,TLS support>> to your application, it is a good idea to add it before re-generating the connector so you only have to test your app once. 
```

Comment:
```suggestion
To add <<tls-support,TLS support>> to your application, add it before re-generating the connector so that you only have to test your app once. 
```